cmd/lncli/cmd_payments.go

```diff
@@ -169,11 +169,11 @@ func paymentFlags() []cli.Flag {
 		},
 		cltvLimitFlag,
 		lastHopFlag,
-		cli.Uint64Flag{
+		cli.Int64SliceFlag{
 			Name: "outgoing_chan_id",
 			Usage: "short channel id of the outgoing channel to " +
 				"use for the first hop of the payment",
-			Value: 0,
+			Value: &cli.Int64Slice{},
```

Comment:
What does this look like on the CLI? like `--outgoing_chan_id=3 --outgoing_chan_id=5`?

Reply:
Yes, it looks like that. Should I indicate in the help section something like `can be used multiple times` to help the user understand?

Reply:
Do you want it to be --outgoing_chan_id=3,4 better? :D

Reply:
We can do it as string divided by comas

Reply:
I think just add "can be used multiple times" is enough.

Reply:
Sorry, was still waking up before :sweat_smile: took a closer look and I think this could be confusing to users as it is phrased right now (can the same channel be used multiple times?). So how about using a semicolon to make it more clear it's a separate sentence and then rephrase to `... payment; can be specified multiple times in the same command`? That also better fits with existing similar flags.

Reply:
Haha don't worry, yeah makes total sense